manifests/deploy/openshift/telemeter/list.yaml

```diff
@@ -0,0 +1,868 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: telemeter-metering
+parameters:
+- name: NAMESPACE
+  value: telemeter-metering-stage
+- name: METERING_VERSION
+  value: 0.11.0
```

Comment:
Let's change this to 4.2

Reply:
Since we're making a var I'll change this to ${METERING_VERSION}